### PR TITLE
Add gatsby source graphql blog

### PR DESCRIPTION
**Pull request by freiksenet**
Depends on #6475

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 35e450435c8bb2177363bc158ed09e6c9390d59c

https://app.netlify.com/sites/using-postcss-sass/deploys/5b4f20e382d3f17c862494f5

**Comment by KyleAMathews** [comment]
Deploy preview for *image-processing* failed.

Built with commit 0a24eb308213d7e7f716a4253a226a687dbc0cc4

https://app.netlify.com/sites/image-processing/deploys/5b51dc1302ed830bc87073b8

**Comment by freiksenet** [comment]
Thanks a lot, I'll do that.

**Comment by m-allanson** [comment]
Nice, LGTM 👍

**Comment by calcsam** [comment]
@freiksenet want to fix the merge conflict here? we'd love to get this in!

**Comment by KyleAMathews** [comment]
@calcsam I'm going to jump on this now and get it ready to get in.

File: docs/blog/author.yaml [hunk 1]
```diff
@@ -156,4 +156,7 @@
 - id: Khaled Garbaya
   bio: Software developer and full-time opensourcerer at @contentful. He speaks multiple languages and is often overheard saying "Bonjour" in HTML.
   avatar: avatars/khaled-garbaya.jpg
-  twitter: "@khaled_garbaya"
```
[the remark this answers is not in the document]
Comment (calcsam) [reply docs/blog/author.yaml]: can't imagine it was. fixed.

**Comment by calcsam** [comment]
Let's get this in before it gets stale again ;) We can always update the date later when we want to display it as recent!

**Comment by KyleAMathews** [comment]
@calcsam I'm still editing this... :-) reverting

**Comment by KyleAMathews** [comment]
Well I guess it's safely buried in the past so I'll finish editing and update the date and tweet it out

**Pull request merged**